google/devtools/clouddebugger/v2/clouddebugger_gapic.yaml

```diff
@@ -0,0 +1,167 @@
+type: com.google.api.codegen.ConfigProto
+language_settings:
+  java:
+    package_name: com.google.cloud.devtools.clouddebugger.spi.v2
+  python:
+    package_name: google.cloud.devtools.clouddebugger.v2
```

Comment:
Wait a min, this reminds me of the error reporting package naming.

In cloud error reporting API, the grpc namespace is somehow `google.devtools.clouderrorreporting`, but the namespace for the Gapic will be different, a bit more manual edit to name them more understandable. See https://github.com/googleapis/googleapis/blob/master/google/devtools/clouderrorreporting/v1beta1/errorreporting_gapic.yaml, the python package names are more like `google.cloud.gapic.errorreporting`.

If that's so, we should do the same thing here as well, omitting `devtools` and `cloud` prefix. Python naming would be `google.cloud.gapic.debugger.v2`,  Ruby `Google::Cloud::Debugger::V2`, node `@google-cloud/debugger`, go `cloud.google.com/go/debugger/apiv2`, java `com.google.cloud.debugger.spi.v2`?  @swcloud @garrettjonesgoogle , @pongad wdyt?